docs/reference/fields.txt

```diff
@@ -1386,8 +1386,39 @@ requiring the usage of the aggregation pipeline for both queries and updates,
 applications should avoid using dots in field names and starting field names
 with the dollar sign if possible.
 
-The Ruby driver `currently prohibits
-<https://jira.mongodb.org/browse/RUBY-2528>`_ inserting documents whose
-field names contain dots or begin with the dollar sign. However, if such
-documents are inserted using other software, Mongoid and the Ruby driver
-provide limited support for retrieving and operating on these documents.
+Mongoid, starting in version 8, now allows users to access fields that begin with
+dollar signs and that contain dots/periods. They can be accessed using the ``send``
+method as follows:
+
+.. code::
+
+    class User
+      include Mongoid::Document
+      field :"first.last", type: String
+      field :"$_amount", type: Integer
+    end
+
+    user = User.first
+    user.send(:"first.last")
+    # => Mike.Trout
+    user.send(:"$_amount")
+    # => 42650000
+
+Since the update operations will not function correctly on these fields, as
+explained above, the setters for these fields are prohibited for use and
+will raise an error:
+
+.. code::
+
+    class User
+      include Mongoid::Document
+      field :"first.last", type: String
+      field :"$_amount", type: Integer
+    end
+
+    user = User.new
+    user.set(:"first.last=", "Shohei.Ohtani")
```

Comment:
Just FYI -- note that `user.set` is an atomic persistence operation which persists directly to the DB. In the future when it is possible to write dot/dollar fields, we should have some way to set the value just on the model, and have it persist when we call `.save!`.

Reply:
This is supposed to be `send` good catch

Reply:
OK. See above comment re: use of `send`.